salt/modules/gpg.py

```diff
@@ -1393,3 +1502,22 @@ def decrypt(
         log.error(result.stderr)
 
     return ret
+
+
+if HAS_GPG_BINDINGS:
+
+    class FixedVerify(gnupg.Verify):
+        """
+        This is a workaround for https://github.com/vsajip/python-gnupg/issues/214.
```

Comment:
Looks like you have pointed to an issue that has been closed and fixed. Do you still need this for newer versions of python-gnupg?

Reply:
It's unnecessary for upcoming versions, but does not interfere with them either. Would you prefer this workaround not be applied to those? (Currently, all released versions that support `sig_info` have this bug since I discovered it working on this feature.)